provisioner/provision_lab.yml

```diff
@@ -166,6 +166,38 @@
         - workshop_type == "networking"
         - ansible_network_os == "ios" or ansible_network_os == "eos"
 
+- name: INSTALL AND CONFIGURE SPLUNK ENTERPRISE
+  hosts: splunk
+  become: true
+  gather_facts: no
+  roles:
+    - role: splunk_enterprise
+      when:
+        - workshop_type == "security_automation"
+
+- name: INSTALL AND CONFIGURE SNORT
+  hosts: snort
+  become: true
+  vars:
+    ids_config_snort_version: '2.9.13'
+    ids_install_provider: "snort"
+    ids_install_normalize_logs: True
+    ids_install_snort_user: root
+    ids_install_snort_group: root
+  tasks:
+    - block:
+      - name: setup epel for snort ecosystem rule lifecycling
+        import_role:
+          name: "geerlingguy.repo-epel"
+      - name: import ids_install role
```

Comment:
Both IDS roles are commited to the repo here as well. Any reason why you do not use roles/requirements.yml?

Reply:
I don't follow.

Reply:
What I mean: you added these roles to this repository. Another way would be to just create a [roles/requirements.yml](https://docs.ansible.com/ansible/latest/reference_appendices/galaxy.html#installing-multiple-roles-from-a-file) file to just "link" to the roles and install them on the managed node when needed via `ansible-galaxy` - instead of them being part of the repository.

The advantages are: less clutter in our repo, and it is easier to keep the roles updated.

Reply:
@liquidat I follow, I'll do that.